src/Routes.js

```diff
@@ -61,21 +60,8 @@ const PackageDetail = lazy(() =>
     )
 );
 
-const Templates = lazy(() =>
-    import(
-        /* webpackChunkName: "Templates" */ './SmartComponents/PatchSet/PatchSet'
-    )
-);
-
-const TemplateDetail = lazy(() =>
-    import(
-        /* webpackChunkName: "TemplateDetail" */ './SmartComponents/PatchSetDetail/PatchSetDetail'
-    )
-);
-
```

Comment:
Can we remove these components since they are no longer used in patch?

Reply:
Yes, this will come in a subsequent PR, I didn't want this one to be so bloated.